crates/proof/proof/src/eip2935.rs

```diff
@@ -3,15 +3,13 @@
 use crate::errors::OracleProviderError;
 use alloc::string::ToString;
 use alloy_consensus::Header;
-use alloy_primitives::{Address, B256, U256, address, b256, keccak256};
+use alloy_eips::eip2935::HISTORY_STORAGE_ADDRESS;
+use alloy_primitives::{B256, U256, b256, keccak256};
 use alloy_rlp::Decodable;
 use alloy_trie::TrieAccount;
 use kona_mpt::{Nibbles, TrieHinter, TrieNode, TrieNodeError, TrieProvider};
 use kona_preimage::errors::PreimageOracleError;
 
-/// The address of the EIP-2935 history storage contract.
-const HISTORY_STORAGE_ADDRESS: Address = address!("0000F90827F1C53a10cb7A02335B175320002935");
-
 /// The [`keccak256`] hash of the address of the EIP-2935 history storage contract.
 const HASHED_HISTORY_STORAGE_ADDRESS: B256 =
```

Comment:
Double checked this as well - `keccak256(0x0000F90827F1C53a10cb7A02335B175320002935) == 0x6c9d57be05dd69371c4dd2e871bce6e9f4124236825bb612ee18a45e5675be51`